### PR TITLE
Typings update

### PR DESCRIPTION
**Pull request by sihu**

### CODE REVIEW

File: typings.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Typings / TS-Lint
+All typings-files are checked in. Make sure you don't have the `typings` in the `.gitignore`.
+
+typings.json:
+```json
+{
+  "name": "<<appname>>", // enter here your appname
```
Comment (lukaselmer) [inline typings.md]: comments = invalid json

File: templates/tslint.json [hunk 2]
```diff
@@ -1,15 +1,33 @@
 {
   "rules": {
+    "align": "todo",
```
Comment (lukaselmer) [inline templates/tslint.json]: todos need to be fixed

File: typings.md (hunk 1 above)
Comment (sihu) [inline typings.md]: so you'd go for ` "name": "<<enter-here-your-appname>>",`

Comment (lukaselmer) [inline typings.md]: aha! uhm, no, i would leave the comments away :)

Comment (lukaselmer) [inline typings.md]: but i will offer you a better solution soon ;)

**Pull request closed without merging**